lesson2/src/config.ts

```diff
@@ -0,0 +1,13 @@
+import logger from './logger';
+
+export const PORT = process.env.PORT || 3000;
+
+logger.info(process.env)
+
+process.on('uncaughtException', (error) => {
```

Comment:
config.ts is not so suitable for error handling

Reply:
Thanks for the comment. I moved both error handlers to the `errorHandlers.ts` file.